ovh/data_cloud_project_containerregistry_oidc.go

```diff
@@ -0,0 +1,103 @@
+package ovh
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceCloudProjectContainerRegistryOIDC() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceCloudProjectContainerRegistryOIDCRead,
+		Schema: map[string]*schema.Schema{
+			"service_name": {
+				Type:        schema.TypeString,
+				Description: "Service name",
+				Required:    true,
+				ForceNew:    true,
+				DefaultFunc: schema.EnvDefaultFunc("OVH_CLOUD_PROJECT_SERVICE", nil),
+			},
+			"registry_id": {
+				Type:        schema.TypeString,
+				Description: "Registry ID",
+				Required:    true,
+				ForceNew:    true,
+			},
+			"delete_users": {
+				Type:     schema.TypeBool,
+				Required: false,
+				Optional: true,
+			},
+			"oidc_name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"oidc_endpoint": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"oidc_client_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"oidc_client_secret": {
```

Comment:
Is it a good practice to expose secrets ?

Reply:
The way to do it is to use the `Sensitive: true` see https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema#Schema

Reply:
Done